docs/android/v2/plugins/Virtual-background.mdx

```diff
@@ -0,0 +1,236 @@
+---
+title: Virtual Background
+nav: 12
+---
+
+## Introduction
+
+Virtual background plugin helps in customising one’s background i.e. replacing the background with a static image.
+This guide provides an overview of usage of the virtual background plugin of 100ms.
+
+![VirtualBackground](/docs/v2/vb.jpg)
+
+## Supported Android API Levels
+
+Min api version for the virtual background plugin is 21. Same as the min api version core sdk for 100ms. The lowest SDK version it can work with is 21. It is built for armeabi-v7a, arm64-v8a, x86, and x86_64 architectures.
```

Comment:
## Supported Devices/Versions

- Minimum Android api level required to support virtual background plugin is 21, same as that required for 100ms SDK

- Minimum 100ms SDK version it can work with is <>

- Virtual background plugin is built for armeabi-v7a, arm64-v8a, x86 and x86_64 architectures